specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2016-12-01/backupManagement.json

```diff
@@ -1107,101 +1104,6 @@
         "deprecated": false
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupJobs": {
```

Comment:
It was buggy on service side which led to broken Python SDK. This change won't break any existing client code using REST APIs directly as old API version is still present in service. Also it won't be a breaking change for SDK users as only serialization of duration is changed in service code which is handled by generated SDKs internally. 

And I don't see how keeping both API versions helps here, Autorest currently doesn't support multiple API versions for same API.